### ISSUE TITLE
Assignees for MergeRequests are "shuffled" after upgrade to 7.10.4

### ISSUE THREAD

**Issue by rob19780114** [gitlabhq/gitlabhq#9332]
After upgrading from 7.7.2 to 7.10.4 our existing projects face a problem where MergeRequests can't be correctly assigned to owners/masters, since they don't show up for the requester.

**Comment by cdaigle** [comment]
If this is the same issue as
https://gitlab.com/gitlab-org/gitlab-ce/issues/1535 it was fixed in 7.11.

**Comment by stanhu** [comment]
Right, I think this was fixed in 7a7ce70 and will be available in a subsequent release.

/cc: @Razer6

**Comment by rob19780114** [comment]
Hi, @stanhu, @cdaigle . Thanks for your input, but unluckily for us it's not fixed. We recently upgraded to 7.11.4 and our problem persists. We were able to trace the problem down, here is our scenario.

We have groups for multiple projects, where usually at least one "owner" and a few "masters" are specified. All projects within the groups are marked as "internal", so every registered user is able to do a fork. The idea was that those forks are merged back to the upstream projects using GitLab's MergeRequest functionality. Again: the projects don't have any users at all; all users are within group.

The odd behaviour is, when adding users with role "Developer" to the projects, they may assign MergeRequests to the original group users, if they are within the same group with the specified role, they don't see anyone.

HTH
Rob
